test/python/test_tools_import_special_phrases.py

```diff
@@ -0,0 +1,174 @@
+
+    _grant_access_to_webuser(temp_db_conn, def_config, phrase_class, phrase_type)
+
+    with temp_db_conn.cursor() as temp_db_cursor:
+        temp_db_cursor.execute(f"""
+                SELECT * FROM information_schema.role_table_grants
+                WHERE table_name='{table_name}' 
+                AND grantee='{def_config.DATABASE_WEBUSER}' 
+                AND privilege_type='SELECT'""")
+        result = temp_db_cursor.fetchone()
+    assert result
+
+@pytest.fixture
+def amenity_operator_funcs(temp_db_cursor):                        
+    temp_db_cursor.execute(f"""
+        CREATE OR REPLACE FUNCTION make_standard_name(name TEXT) RETURNS TEXT
+        AS $$
+        DECLARE
+        o TEXT;
+        BEGIN
+        RETURN trim(name); --Basically return only the trimed name for the tests
+        END;
+        $$
+        LANGUAGE plpgsql IMMUTABLE;
+
+        CREATE SEQUENCE seq_word start 1;
+
+        CREATE OR REPLACE FUNCTION getorcreate_amenity(lookup_word TEXT, normalized_word TEXT,
+                                                    lookup_class text, lookup_type text)
+        RETURNS INTEGER
+        AS $$
+        DECLARE
+        lookup_token TEXT;
+        return_word_id INTEGER;
+        BEGIN
+        lookup_token := ' '||trim(lookup_word);
+        SELECT min(word_id) FROM word
+        WHERE word_token = lookup_token and word = normalized_word
+                and class = lookup_class and type = lookup_type
+        INTO return_word_id;
+        IF return_word_id IS NULL THEN
+            return_word_id := nextval('seq_word');
+            INSERT INTO word VALUES (return_word_id, lookup_token, normalized_word,
+                                    lookup_class, lookup_type, null, 0);
+        END IF;
+        RETURN return_word_id;
+        END;
+        $$
+        LANGUAGE plpgsql;
+
+        CREATE OR REPLACE FUNCTION getorcreate_amenityoperator(lookup_word TEXT,
+                                                       normalized_word TEXT,
+                                                       lookup_class text,
+                                                       lookup_type text,
+                                                       op text)
+        RETURNS INTEGER
+        AS $$
+        DECLARE
+        lookup_token TEXT;
+        return_word_id INTEGER;
+        BEGIN
+        lookup_token := ' '||trim(lookup_word);
+        SELECT min(word_id) FROM word
+        WHERE word_token = lookup_token and word = normalized_word
+                and class = lookup_class and type = lookup_type and operator = op
+        INTO return_word_id;
+        IF return_word_id IS NULL THEN
+            return_word_id := nextval('seq_word');
+            INSERT INTO word VALUES (return_word_id, lookup_token, normalized_word,
+                                    lookup_class, lookup_type, null, 0, op);
+        END IF;
+        RETURN return_word_id;
+        END;
+        $$
+        LANGUAGE plpgsql;""")
```

Comment:
If you want, you can greatly simplify these function to just do a dummy action and/or return a dummy result. The SQL is not under test here. The test just need to check that the functions were called at all.